### PR TITLE
MCP-1814: remove PII from Controller logs

### PR DESCRIPTION
**Pull request by dimitri-amida**
# Feature Name 

### What was the problem?

Some PII was present in log statements

### How does this fix it?

Removed PII

### Jira Tickets

- [MCP-1814](https://amida.atlassian.net/browse/MCP-1814)

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

File: controller/src/main/java/gov/va/vro/controller/VroController.java [hunk 1]
```diff
@@ -123,14 +126,17 @@ public ResponseEntity<Object> fetchPdf(String claimSubmissionId)
   public ResponseEntity<FullHealthDataAssessmentResponse> postFullHealthAssessment(
       HealthDataAssessmentRequest claim)
       throws RequestValidationException, ClaimProcessingException {
-    log.info("Getting health assessment for: {}", claim.getVeteranIcn());
+    log.info("Getting health assessment for veteran icn: {}", claim.getVeteranIcn());
     try {
       Claim model = postClaimRequestMapper.toModel(claim);
       String responseAsString = camelEntrance.submitClaimFull(model);
-      log.info("Obtained full health assessment: {}", responseAsString);
```
Comment (dimitri-amida) [inline controller/src/main/java/gov/va/vro/controller/VroController.java:130]: contains PII

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java [hunk 2]
```diff
@@ -36,15 +34,6 @@ public void configure() {
     configureRouteFetchPdf();
   }
 
-  private void configureRouteFileLogger() {
```
Comment (dimitri-amida) [inline service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java:39]: Not used for anything

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java [hunk 3]
```diff
@@ -26,8 +27,8 @@ public class SlipClaimSubmitRouter {
   public String routeClaimSubmit(Object body, @ExchangeProperties Map<String, Object> props) {
     Object diagnosticCodeObj = props.get("diagnosticCode");
     if (diagnosticCodeObj == null) {
-      log.error("No diagnostic code in the body.");
-      return null;
+      log.error(NO_DIAGNOSTIC_CODE_ERROR);
+      throw new CamelProcessingException(NO_DIAGNOSTIC_CODE_ERROR);
```
Comment (dimitri-amida) [inline service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java:31]: throw exception to avoid an NPE on the controller

Comment (wlin2019) [reply service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java:31]: The compiler is okay with this?

**Review by wlin2019 — APPROVED** [review]
I haven't tested it. Please test it to make sure it works.

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java (hunk 3 above)
Comment (dimitri-amida) [reply service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java:31]: Yes, why would it not be?

**Review by ianwolson — APPROVED** [review]

**Pull request merged**